src/components/query-viewer-pivot-table/controller/query-viewer-pivot-render.tsx

```diff
@@ -285,24 +300,40 @@ export class QueryViewerPivotTableRender {
   }
 
   render() {
-    if (this.serviceResponse == null) {
-      return "";
-    }
-
-    const pivotParameters = this.getPivotTableParms();
+    const pivotParameters = this.getPivotTableParameters();
```

Comment:
It's easier to understand that if we don't have the metadata and data to render the output (`serviceResponse == null`), we must not render anything. Also, if we did render and output in a certain time, but then the `serviceResponse == null`, we would keep the previous render instead of destroying it.

```suggestion
    if (this.serviceResponse == null) {
      return "";
    }

    const pivotParameters = this.getPivotTableParameters();
```